src/clj/game/core/events.clj

```diff
@@ -93,13 +93,15 @@
                 (if (or (= 1 (count handlers)) (empty? interactive) (= 1 (count non-silent)))
                   (let [to-resolve
                         (if (= 1 (count non-silent)) (first non-silent) (first handlers))
+                        ab (dissoc (:ability to-resolve) :req)
+                        c (:card to-resolve)
```

Comment:
Could these be called `ab-to-resolve` or `ability-to-resolve` and/or `card-to-resolve`? Just preference, but I like more explicit names.